onnx/reference/reference_evaluator.py

```diff
@@ -294,6 +310,24 @@ def __init__(  # type: ignore
                 self.new_ops_[key] = cl
         self._init()
 
+    def retrieve_external_data(self, init: TensorProto) -> np.array:
+        """
+        Returns a tensor saved as external.
+        """
+        info = ExternalDataInfo(init)
+        location = info.location
+        if location.startswith("#"):
```

Comment:
Can this check be provided via an api from the large container module?

Reply:
Done, I added method `is_in_memory_external_initializer`.